### PR TITLE
github sha on merge for develop repo

### PR DESCRIPTION
**Pull request by jterryfr**
Create github sha in the develop repo.

issue:  https://github.com/SecureBankingAccessToolkit/SecureBankingAccessToolkit/issues/89

### CODE REVIEW

File: .github/workflows/merge.yml [hunk 1]
```diff
@@ -21,4 +21,5 @@ jobs:
       
       - name: docker build
         run: |
-          make build-docker gcr-repo=${{ secrets.RELEASE_REPO }}
+          make build-docker-ig tag=${{ github.sha }}
```
Comment (jorgesanchezperez) [inline .github/workflows/merge.yml:24]: This will build only Ig, that's correct?

**Review by BohoCode — APPROVED** [review]

File: .github/workflows/merge.yml (hunk 1 above)
Comment (jorgesanchezperez) [reply .github/workflows/merge.yml:24]: It won't be better something like below in the make file:
`docker build docker/7.0/ig/ -t eu.gcr.io/${gcr-repo}/securebanking/gate/ig:latest -t eu.gcr.io/${gcr-repo}/securebanking/gate/ig:${tag}`
and
`docker push ....`

And call the make file only once `make build-docker-ig tag=xxx gcr-repo=xxx`

**Review by jorgesanchezperez — COMMENTED** [review]
some comments added.

File: .github/workflows/merge.yml [hunk 2]
```diff
@@ -21,4 +21,6 @@ jobs:
...
+          docker tag eu.gcr.io/${{ secrets.DEV_REPO}}:{{ github.sha }} eu.gcr.io/${{ secrets.RELEASE_REPO}}:latest
+          docker push eu.gcr.io/${{ secrets.RELEASE_REPO}}:latest
```
Comment (jorgesanchezperez) [inline .github/workflows/merge.yml:26]: Must be the same repository to have the same image tagged with latest and github.sha.
- docker push eu.gcr.io/${REPO}:latest
- docker push eu.gcr.io/${REPO}:{{ github.sha }}

**Review by jorgesanchezperez — CHANGES_REQUESTED** [review]

File: .github/workflows/merge.yml [hunk 3]
```diff
@@ -21,4 +21,6 @@ jobs:
       
       - name: docker build
         run: |
-          make build-docker gcr-repo=${{ secrets.RELEASE_REPO }}
+          make build-docker-ig tag=${{ github.sha }}
+          docker tag eu.gcr.io/${{ secrets.DEV_REPO }}:{{ github.sha }} eu.gcr.io/${{ secrets.DEV_REPO }}:latest
+          docker push eu.gcr.io/${{ secrets.DEV_REPO }}:latest
```
[the remark this answers is not in the document]
Comment (jterryfr) [reply .github/workflows/merge.yml]: the initial `make build-docker-ig tag=${{ github.sha }}` already does that

**Review by jorgesanchezperez — APPROVED** [review]

**Pull request merged**